src/main/java/mate/academy/bookstore/controller/BookController.java

```diff
@@ -44,7 +45,7 @@ public void delete(@PathVariable Long id) {
     }
 
     @PostMapping
-    public BookDto createBook(@RequestBody CreateBookRequestDto book) {
+    public BookDto createBook(@RequestBody @Valid CreateBookRequestDto book) {
```

Comment:
Didn't find update book endpoint, please add it

Reply:
comment not fixed

Reply:
![image](https://github.com/user-attachments/assets/a7ad526d-d170-42f6-9797-a2a64115497e)

Reply:
There is a Valid annotation

Reply:
I didn't text about Valid annotation anything. 
BookController doesn't have update endpoint, so please add it